docs/UserGuide.md

```diff
@@ -3,12 +3,44 @@ layout: page
 title: User Guide
 ---
 
-RealEstatePro is a desktop app for managing contacts, optimized for real estate agents to manage their client's contacts and sales of properties.
+RealEstatePro is a desktop app for managing contacts, optimized for real estate agents to manage their client's contacts and sales of HDB properties.
+The app also includes various tools which aims to help agents in their work.
 
 # Reference
 
 Original AB3 User Guide: [link](https://se-education.org/addressbook-level3/UserGuide.html)
 
+#Table of contents
+- [Quick Start](#quick-start)
+- [Features](#features)
+- [General Commands](#general-commands)
+  - [Open Help Window: `help`](#open-help-window-help)
+  - [Listing all clients: `list`](#listing-all-clients--list)
+  - [Clearing all entries: `clear`](#clearing-all-entries--clear)
+  - [Exiting the program: `exit`](#exiting-the-program--exit)
+- [Client Management](#client-management)
```

Comment:
Great work adding these valuable headers. Makes the app have an even more 'for real estate agents' feel to it!